code/game/objects/effects/effect_system.dm

```diff
@@ -415,28 +415,12 @@ steam.start() -- spawns the effect
 
 /obj/effect/effect/chem_smoke/New()
 	..()
-	var/datum/reagents/R = new/datum/reagents(500)
-	reagents = R
-	R.my_atom = src
-
 	spawn (200+rand(10,30))
 		delete()
 	return
 
 /obj/effect/effect/chem_smoke/Move()
 	..()
-	for(var/atom/A in view(2, src))
-		if(reagents.has_reagent("radium")||reagents.has_reagent("uranium")||reagents.has_reagent("carbon")||reagents.has_reagent("thermite")||reagents.has_reagent("synthflesh")||reagents.has_reagent("firefighting_foam")||reagents.has_reagent("honey"))//Prevents unholy radium spam by reducing the number of 'greenglows' down to something reasonable -Sieve

```

Comment:
should be able to uncomment honey, synth flesh, and a few other reaction_turfs now.